domain/src/main/java/org/oppia/android/domain/question/QuestionConstantsProvider.kt

```diff
@@ -11,6 +11,18 @@ annotation class QuestionCountPerTrainingSession
 @Qualifier
 annotation class QuestionTrainingSeed
 
+@Qualifier
+annotation class ViewHintScorePenalty
```

Comment:
Each of these qualifiers should have documentation to explain what they correspond to/how to use them. E.g.:

```kotlin
/** Qualifier corresponding to the penalty users receive for each hint viewed in a practice session (an application-level injectable int). */
@Qualifier annotation class ViewHintScorePenalty
```

Reply:
Added, thanks!